.dockstore.yml

```diff
@@ -55,3 +55,10 @@ workflows:
     readMePath: /wdl/ExportDataFromDataset/README.md
     testParameterFiles:
       - /wdl/ExportDataFromDataset/ExportDataFromDataset.wdl
+
+  - name: HardCloneWorkspace
```

Comment:
Does "hard" clone (as opposed to some other type of cloning) mean anything to users? I'm wondering if for them we just display this as `CloneWorkspace` ?

Reply:
In Terra they do have the option to clone a workspace so they might familiar with that. I am open to naming it either way though if it makes more sense.